frame/bounties/src/tests.rs

```diff
@@ -142,10 +139,8 @@ impl Config for Test {
 	type MaximumReasonLength = MaximumReasonLength;
 	type WeightInfo = ();
 }
-type System = frame_system::Module<Test>;
-type Balances = pallet_balances::Module<Test>;
-type Treasury = pallet_treasury::Module<Test>;
-type Bounties = Module<Test>;
+
+type TreasuryEvent = pallet_treasury::Error::<Test, pallet_treasury::DefaultInstance>;
```

Comment:
This should be renamed TreasuryError

Reply:
Er yeah just checking you were paying attention :sweat_smile: